echopype/calibrate/calibrate_ek.py

```diff
@@ -311,7 +320,7 @@ def __init__(self, echodata, env_params, cal_params, waveform_mode, encode_mode)
         # self.range_meter computed under self._compute_cal()
         # because the implementation is different depending on waveform_mode and encode_mode
 
-    def get_env_params(self, env_params, waveform_mode=None, encode_mode="complex"):
+    def get_env_params(self, env_params, waveform_mode="BB", encode_mode="complex"):
         """Get env params using user inputs or values from data file.
 
         EK80 file by default contains sound speed, temperature, depth, salinity, and acidity,
```

Comment:
acidity, really?? Requiring a pH sensor seems surprising

Reply:
The sound speed equations require acidity and the EK80 allows the users to provide it. Most (all?) people just set it to 8.0.

Reply:
Interesting, thanks